community/kernel/src/main/java/org/neo4j/kernel/impl/pagecache/ConfiguringPageCacheFactory.java

```diff
@@ -126,12 +126,13 @@ public int calculatePageSize( Config config, PageSwapperFactory swapperFactory )
 
     public void dumpConfiguration()
     {
+        int MiB = 1024 * 1024;
```

Comment:
We already have corresponding constants in `Format`, even if `Format` is not really expected it, lets still use those.